### ISSUE TITLE
Mehr Bilder einfügen

### ISSUE THREAD

**Issue by krother** [veit-schiele-communications/python-fuer-einsteiger#1]
mindestens ein aussagekräftiges Leitbild / Übersicht pro Kapitel.

**Comment by veit** [comment]
Soll sich André das mal anschauen? Er ist am Freitag im Büro und falls Du Zeit hast, könnten wir uns dann auch gemeinsam treffen. Was hältst Du davon?

**Comment by krother** [comment]
ja gerne, bin am Freitag ab 12 Uhr allerdings an der Uni. Ich könnte für André auch ein paar Beispiele zusammenstellen, damit er weiß, was ich selbst mit Inkscape hinbekomme.

**Comment by veit** [comment]
Die Beispiele wären sicherlich hilfreich um ihm den Einstieg in die Thematik zu erleichtern. Hättest Du ggf. nach der Uni noch Zeit und Lust im Büro vorbeizukommen?

**Comment by krother** [comment]
Ja, könnte um 18 Uhr da sein. Wäre für mich viel einfacher als das Vormittags einzuquetschen.
